app/javascript/components/top-navbar/notifications.jsx

```diff
@@ -17,7 +17,7 @@ const Notifications = () => {
       <a
         id="notifications-btn"
         className="nav-item-iconic drawer-pf-trigger-icon"
-        title={`${unreadCount} ${__('unread notifications')}`}
+        title={sprintf(__('%d unread notifications'), unreadCount)}
```

Comment:
I have a feeling that some i18n is missing from here, cc @mzazrivec

Reply:
Good catch, fixed :)

Reply:
..And fixed the tests that failed because of that :D